metrics/helpers/lp.py

```diff
@@ -123,6 +123,57 @@ def get_team_daily_triage_count(team, distro, blacklist=None):
     return len(results)
 
 
+def get_team_unassigned_bugs(team, distro):
+    """
+    Report count of new unassigned bugs for Launchpad team on a distro.
+
+    List the number of bugs that are New and unassigned for a particular
+    team subscribed to the bugs.
+    """
+    lp_distro = LP.distributions[distro]
+    lp_team = LP.people[team]
+    return len(lp_distro.searchTasks(bug_subscriber=lp_team, assignee=None,
+                                     status='New'))
+
+
+def get_team_incomplete_bugs(team, distro):
```

Comment:
See my other comment about renaming the function to make it clearer that bug_subscriber is the filter here.